src/Resources/public/core.js

```diff
@@ -941,20 +941,24 @@ var Backend =
 				alert('Could not find the SimpleModal frame');
 				return;
 			}
-			ul = frm.document.getElementById(opt.id);
+			ul = frm.document.getElementById('tl_select');
```

Comment:
Hm, the core is not passing an `id` attribute anymore, because you are supposed to pass a callback function instead. Therefore, `tl_select` is kind of a convention.

Is it crucial that we support `opt.id` here, too?